MODULE.bazel

```diff
@@ -17,3 +20,34 @@ archive_override(
     strip_prefix = "gz-utils-gz-utils3",
     urls = ["https://github.com/gazebosim/gz-utils/archive/refs/heads/gz-utils3.tar.gz"],
 )
+
+PYTHON_VERSIONS = [
+    "3.8",
```

Comment:
FYI Python 3.8 is EOL https://devguide.python.org/versions/

Reply:
removed